common/djangoapps/student/tests/test_email.py

```diff
@@ -132,6 +135,26 @@ def _assert_activation_email(self, subject, body_fragments):
         for fragment in body_fragments:
             self.assertIn(fragment, msg.body)
 
+    @mock.patch('student.tasks.log')
+    def test_send_email_to_inactive_user(self, mock_log):
+        """
+        To verify that activation email has been sent to
+        an un-activated user(logged-in via social-auth)
+        """
+        self.inactive_user = UserFactory(is_active=False)
+        self.request_factory = RequestFactory()
```

Comment:
These variables `inactive_user` , `request_factory` used only within the function. So you don't need to use with the `self`. if you wanted to do so, then define it outside the function.